faw/pdf-observatory/common/dsl.ts

```diff
@@ -5,14 +5,22 @@ import peg from 'pegjs';
 export const dslDefault = `# Loading...`;
 
 export interface DslResult {
+  extraFeatures: Array<DslExtraFeature>;
   filters: Array<DslFilter>;
   outputs: Map<string, DslOutput>;
 }
 
+export interface DslExtraFeature {
```

Comment:
There's an argument that we should/could reuse the `DslFilter` type for this, but I think calling out the difference between `DslFilter.name` (not matched by regexes) and `DslExtraFeature.featureText` (matched by regexes) is important enough to warrant a separate type.